tests/nodeLabeller_test.go

```diff
@@ -1,222 +0,0 @@
-package tests
```

Comment:
Instead of removing this file, can you modify it to test that the `node-labeller` resources do not exist? It will be important for upgrade test lane.